ci/azure-pipelines.yml

```diff
@@ -54,12 +54,15 @@ jobs:
 
   - job: IntegrationTests
     pool:
-      vmImage: ubuntu-16.04
-    timeoutInMinutes: 120
+      vmImage: ubuntu-18.04
+    strategy:
+      parallel: 5
```

Comment:
question wrt azp - will this consume overall parallel job quota?

Reply:
It does, but the majority of the parallel jobs finish in a rather short period, if we see that this is creating a backlog we can always adjust this number lower, but I don't expect it to happen.